web/web/auth/static/actions.js

```diff
@@ -1,6 +1,95 @@
+import { api } from "../../static/js/network_client";
+import { menuRoutes } from "../../static/js/config/routes";
+import { createLoginError } from "./helpers";
+
 export function loginSuccessful(token) {
+    window.location.href = menuRoutes[0].path;
```

Comment:
Actions should be pure functions that return only JS objects, unless it's a [thunk](https://github.com/reduxjs/redux-thunk). In this case, the successful login should update the state in the reducer and the react component should update it's state (bc it's connected to this reducer) and subsequently redirect the user. You should be able to just move this to the auth component.